### PR TITLE
Migrate /privacy pages to Fluent

### PR DESCRIPTION
**Pull request by maxxcrawford**
## Description
Updates the contact section of the privacy pages _(completed by @craigcook)_  and migrates the pages to Fluent in the process. 

- Privacy (/privacy/)
- Privacy FAQs (/privacy/faq)
- Privacy Principles (/privacy/principles/)

## Issue / Bugzilla link

- #9388 
- #9809

## Testing

- http://localhost:8000/privacy/
- http://localhost:8000/privacy/principles/
- http://localhost:8000/privacy/faq/

### CODE REVIEW

File: bedrock/privacy/templates/privacy/index.html [hunk 1]
```diff
@@ -33,120 +33,57 @@
...
-          <span itemprop="addressLocality">Mountain View</span>,
-          <span itemprop="addressRegion">CA</span>
-          <span itemprop="postalCode">94041</span><br>
-          <span itemprop="email"><a href="mailto:compliance@mozilla.com">compliance@mozilla.com</a></span>
-        </span>
-      </p>
+    <p lang="en" dir="ltr" itemscope itemtype="http://schema.org/Organization">
+      <span itemprop="name">Mozilla Corporation</span><br>
+      <span itemprop="address" itemscope itemtype="http://schema.org/PostalAddress">
+        {{ _('Attn:') }} <span itemprop="name">Mozilla - Privacy</span><br>
```
Comment (alexgibson) [inline bedrock/privacy/templates/privacy/index.html:42]: Nit: unwrap old-format string if it's hard-coded?
```suggestion
        Attn: <span itemprop="name">Mozilla - Privacy</span><br>
```

File: bedrock/privacy/templates/privacy/index.html [hunk 2]
```diff
@@ -33,120 +33,57 @@
...
+        <span itemprop="streetAddress">2 Harrison St., # 175</span>,<br>
+        <span itemprop="addressLocality">San Francisco</span>,
+        <span itemprop="addressRegion">CA</span>
+        <span itemprop="postalCode">94105</span><br>
+        <span itemprop="addressCountry">USA</span><br>
+        <span itemprop="email"><a href="mailto:compliance@mozilla.com">compliance@mozilla.com</a></span>
+      </span>
+    </p>
 
-    {% if l10n_has_tag('privacy_email_2019') %}
```
Comment (alexgibson) [inline bedrock/privacy/templates/privacy/index.html:57]: Some of the L10n conditionals in this file go back as far as 2019 / 2018. Should we consider removing some of the older ones to try and tidy up the template a little? Locales have had a long time to catch up, so maintaining all of these may no longer really be necessary.

**Review by alexgibson — APPROVED** [review]
A couple of minor suggestions but nothing that should block merging. Nice work! 💯

**Comment by craigcook** [comment]
Updated! I left the conditionals around the sidebar blocks, but we could remove those as well and just show mixed content for the locales that still don't have them translated.

**Pull request merged**